code/game/objects/structures/fireaxe.dm

```diff
@@ -111,16 +110,15 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/fireaxecabinet, 32)
 	. = ..()
 	if(.)
 		return
-	if(open || broken)
-		if(fireaxe)
-			user.put_in_hands(fireaxe)
-			fireaxe = null
-			to_chat(user, span_notice("You take the fire axe from the [name]."))
-			src.add_fingerprint(user)
-			update_appearance()
-			return
+	if((open || broken) && fireaxe)
+		user.put_in_hands(fireaxe)
+		to_chat(user, span_notice("You take [fireaxe] from [src]."))
+		fireaxe = null
+		src.add_fingerprint(user)
+		update_appearance()
+		return
 	if(locked)
-		to_chat(user, span_warning("The [name] won't budge!"))
+		to_chat(user, span_warning("\The [src] won't budge!"))
```

Comment:
pretty sure [src] automatically includes \the

Reply:
Will that respect the need for capitalizing "The" as the first letter of a sentence?